src/client/components/Input/Input.css

```diff
@@ -0,0 +1,34 @@
+input {
```

Comment:
You are using a global selector `input` and you should not do that. Switch to class.